apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -1031,7 +1031,7 @@ <h3 class="ledgerHead">
                                         {{ ledgerTxnBalance?.currencySymbol }}{{lc.reckoningCreditTotal | giddhCurrency }}
                                     </span>
 
-                                    <span class="bdrT" *ngIf="isLedgerAccountAllowsMultiCurrency"
+                                    <span class="bdrT text-gray" *ngIf="isLedgerAccountAllowsMultiCurrency"
```

Comment:
remove inline CSS everywhere

Reply:
what means everywhere? in this file i am not able to see any inline css

Reply:
i am not added any inline css